src/main/java/org/cbioportal/persistence/mybatis/StudyMyBatisRepository.java

```diff
@@ -16,14 +16,12 @@ public class StudyMyBatisRepository implements StudyRepository {
 
     @Autowired
     private StudyMapper studyMapper;
-    @Autowired
-    private OffsetCalculator offsetCalculator;
 
     @Override
     public List<CancerStudy> getAllStudies(String keyword, String projection, Integer pageSize, Integer pageNumber,
                                            String sortBy, String direction) {
 
-        return studyMapper.getStudies(null, keyword, projection, pageSize, offsetCalculator.calculate(pageSize, pageNumber), 
+        return studyMapper.getStudies(null, keyword, projection, pageSize, PaginationCalculator.offset(pageSize, pageNumber), 
```

Comment:
Once previous comment is addressed

PaginationCalculator::offset(pageSize, pageNumber)